### ISSUE TITLE
Screen reader not announcing aria-valuemin, aria-valuemax for Slider component

### ISSUE THREAD

**Issue by irakaundal98** [microsoft/fluentui#18116]
### Environment Information

- **Package version(s)**: Taken from documentation: https://developer.microsoft.com/en-us/fluentui#/controls/web/slider
- **Browser and OS versions**: Chrome (90.0.4430.93 (Official Build) (x86_64)). 
- OS - Version	10.0.21364 Build 21364

### Describe the issue:

Screen readers do not announce the aria-valuemin, aria-valuemax property for Slider.

I have tried the screen reader extension on Chrome. It announced the min, max value for other examples but not for the fluent ui control.

Narrator is not announcing min, max value in both cases. Narrator should support announcing these values as well.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/irakaundal/pen/BaWNOxd

#### Actual behavior:

Screen reader doesn't announce the aria-valuemin and aria-valuemax property for the slider.

#### Expected behavior:

Screen reader should announce the aria-valuemin and aria-valuemax property.

### Documentation describing expected behavior

https://www.w3.org/TR/wai-aria-practices-1.1/#slider

**Comment by TristanWatanabe** [comment]
@irakaundal98: The behavior you're looking for seems to be screen reader specific. While I understand the potential use case for having the min and max announced by the screen reader, this behavior isn't something we have control over as we're already providing the necessary `aria-valuemin` and `aria-valuemax` attributes for screen readers to consume. Another issue here is that we're using a `div` to implement a range slider as opposed to the native `input type= "range"` control which may be why the min and max are not being read out by chrome's browser screen reader. In this case, fixing would require significant DOM changes which is not something we'll be able to make at this time. With all that being said, I'll have to close this issue since there's nothing more we can do for now.
